expressions_api/expression_verify.go

```diff
@@ -18,22 +19,49 @@ const THREAD_POOL = 500
 type ExpressionVerificationAccessor interface {
 	//Batch API which parallelizes expression verification calls
 	VerifyExpressions(ctx context.Context, verifyExpressionsInput internal.VerifyExpressionsInput) internal.VerifyExpressionsOutput
+	RefreshSpannerClient(ctx context.Context, project string, instance string) error
```

Comment:
I'm assuming the changes to call the VerifyExpressions will be in next PRs. But, I thought RefreshSpannerClient will happen inside `VerifyExpressions` method and the user need not call RefreshSpannerClient before calling VerifyExpressions. 

Either design is fine, I just want to make sure this is intentionally done this way.

Reply:
Yes this was intentional as this ends up only being called during ProcessSchema(when the client is used again), I didn't want to increase redundancy of calling this is session creation is expensive